doc/sources/patching/patching-options.rst

```diff
@@ -28,6 +28,10 @@
     from sklearnex import patch_sklearn
     patch_sklearn()
```

Comment:
Could you please also add note here that it's required to reimport in this case? 

And also same not to main readme.md